app/population/controllers/edit.js

```diff
@@ -57,5 +58,15 @@ class DetailsController extends FormWizardController {
 
     next()
   }
+
+  setCancelUrl(req, res, next) {
+    if (req.population) {
```

Comment:
Could shorten this using a ternary to remove duplication:

```javascript
res.locals.cancelUrl = `/population/day/${req.date}${req.population ? `/${req.locationId}` : ''}`
```

Reply:
@teneightfive One of the paths uses `/day` and the other uses `/week` - it's clearer if they're kept separate.

Also, the "if population, edit, else, create" pattern is used on the rest of the population route, so it makes sense to keep to it

Reply:
Gotcha 👍